<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>349</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Field size</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="xSizeSpinBox">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="xSizeSlider">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="ySizeSpinBox">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="ySizeSlider">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QSpinBox" name="speedSpinBox">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="speedSlider">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Cell size</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QSpinBox" name="cellSizeSpinBox">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>27</height>
                </size>
               </property>
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="cellSizeSlider">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>18</height>
                </size>
               </property>
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="replaceRadioButton">
               <property name="text">
                <string>Replace</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="xorRadioButton">
               <property name="text">
                <string>Xor</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="impactCheckBox">
             <property name="text">
              <string>Impact</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>FIRST_IMPACT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="firstImpactSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>SECOND_IMPACT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="secondImpactSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>LIVE_BEGIN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="liveBeginSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>LIVE_END</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="liveEndSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>BIRTH_BEGIN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="birthBeginSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>BIRTH_END</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="birthEndSpinBox">
             <property name="minimumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>62</width>
               <height>27</height>
              </size>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ySizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ySizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cellSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cellSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
